week_6/week_6_quiz.md

```diff
@@ -0,0 +1,251 @@
+The ABCD substance use interview includes questions about: (check all that apply)
+
+- [ ] which substances youth have heard of
+- [ ] subjective effects of early use (alcohol, nicotine, cannabis)
+- [ ] Problem symptoms related to use (alcohol, nicotine, cannabis)
+- [ ] peer substance use and availability of substances in the neighborhood
+- [ ] parent substance use and family history of use
+- [ ] caffeine consumption and over-the-counter medications
+
+
+**Question 2**
+
+The goals of the ABCD substance use module include the characterization of: (check all that apply)
+
+- [ ] risk and resilience factors for substance use and SUD development (e.g., early substance use attitudes, expectancies, subjective effects, peer use, parental rules)
+- [ ] the impact of polysubstance use patterns and “gateway interactions” between different substances
+- [ ] the consequences of repeated substance use on neurocognitive, emotional, and physical development
+- [ ] associations between substance use and psychopathology
+- [ ] differential trajectories of substance use initiation, experimentation, and use patterns
+
+
+**Question 3**
+
+Biospecimens (e.g., saliva, hair, and urine samples) provide a more objective measure of recent substance use and exposure. Biospecimen samples are collected for ALL participants enrolled in the ABCD Study at each in-person visit.
+
+- True
+- False
+
+**Question 4**
+
+Which of the following are examples of gating in the ABCD substance use interview? (check all that apply)
+
+- [ ] In years baseline-2 year follow-up, a participant indicates they have “heard of” a given substance, additional questions about that particular substance will follow
+- [ ] If a participant indicates they have heard of a potential “gateway” substance, additional questions about other substances will follow
+- [ ] If a participant indicates they have NOT used a given substance, zeros may be inserted for any follow-up questions triggered by this question
+- [ ] If a participant has indicated they have used a given substance, they will be asked to recall details of their first experience with that substance at future visits
+
+
+**Question 5**
+
+The Timeline Follow Back asks about detailed month-by-month substance use (i.e., date of use, amount in standard units) and applicable follow-up questions for some substances such as cannabis (e.g., route of administration) for the 12-month period preceding the interview date.
+
+- True
+- False
+
+
+### ReproNim NIDM data questions
+
+#### Prerequisites
+
+1. You have already [forked](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo) and [cloned](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/cloning-a-repository): https://github.com/ABCD-ReproNim/sample_dataset
+2. You have installed [pynidm](https://pypi.org/project/pynidm/) and [pyontutils](https://pypi.org/project/pyontutils/) (or are using our jupyterhub)
```

Comment:
I'd point to the docs sites (which typically have installation instructions) instead of PyPi. There's not generally a lot of useful info on PyPi.

Reply:
I updated the pynidm link, but I couldn't find any formal documentation for pyontutils, at least the pypi page has how to pip install install the package at the top of the page.